dockerfiles/mingw-fedora/Dockerfile

```diff
@@ -0,0 +1,55 @@
+FROM fedora:30
+
+# Update base system
+RUN dnf update -y
+
+# Add some useful packages to the base system
+RUN dnf install -y \
+        make \
+        cmake \
+        git \
+        sudo \
+        wget \
+        gawk \
+        sudo \
+        fakeroot \
+        file \
+        patch \
+        ninja-build
+
+# Install MingW packages
+RUN dnf install -y \
+        mingw32-binutils \
+        mingw32-boost \
+#        mingw32-cmake \
+        mingw32-crt \
+        mingw32-curl \
+        mingw32-fontconfig \
+        mingw32-freetype \
+        mingw32-gcc \
+        mingw32-headers \
+        mingw32-jansson \
+        mingw32-libpng \
+        mingw32-libzip \
+        mingw32-openssl \
+        mingw32-pkg-config \
+        mingw32-SDL2 \
+        mingw32-SDL2_ttf \
+        mingw32-speexdsp \
+        mingw-w64-tools \
+        mingw32-winpthreads \
+        mingw32-win-iconv \
+        mingw32-zlib
+
+RUN useradd --create-home --comment "Fedora Build User" build && \
+    groupadd sudo && \
+    echo '%sudo ALL=(ALL) NOPASSWD:ALL' >> /etc/sudoers; \
```

Comment:
I don't see a reason to use `sudo`, please explain

Reply:
I did a basic copy from the mingw-arch, it was in there, so I kept it. But you are probably right as I think it was used to compile extra packages that aren't in the arch package repository.